packages/stacked_tools/lib/src/services/config_service.dart

```diff
@@ -64,6 +64,12 @@ class ConfigService {
         _customConfig.testViewsPath,
       );
 
+  /// Returns boolean value to determine view builder style.
+  ///
+  /// False: ViewModelWidget
+  /// True: ViewModelBuilder
+  bool get useViewModelBuilderStyle => _customConfig.useViewModelBuilderStyle;
```

Comment:
Oooooooh. I'm only seeing now. This is in the stacked_config. Good choice! We should be able to override as well from the command.

Reply:
I will add the override through command.